src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -293,6 +308,10 @@ The `navigate` method takes these arguments:
 
 For backward and forward navigation through the browser history, you can combine `navigate()` with the built-in `history.back()`, `history.forward()` and `history.go()` functions of the browser.
 
+:::note
```

Comment:
I think this note might flow better right after the React example, and maybe include a very short code snippet of a `.astro` file rendering a `<React client:load />` directive? Then, they are getting this information *while* seeing about UI framework components, and also seeing usage of the component. What do you think?

I'm also not convinced it should be a *note*, since it's just how stuff works. But, let's at least see what we think about moving it up first. And visually there, maybe a note would look too overwhelming with all the code samples anyway, and regular text would be just fine! (since there's less plain text to distinguish it from)

Reply:
Thanks! The fewer notes the better.

There was a mention of client components already, I've edited it to clarify.